### PR TITLE
Bugs/refactor stuff

### PR DESCRIPTION
**Pull request by m4rtint**
This one is a bit insane. 
But it's 
- Code adding namespaces
- Code adding = null
- Organizing files
- Changed - StoredString -> Data, Placed all stored strings into their respective structs

### CODE REVIEW

**Comment by m4rtint** [comment]
[Refactor Project](https://trello.com/c/2XEtmRTK/422-refactor-project)

File: Assets/_PartyParkade/Script/Data/CommonAnimations.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="CommonAnimations.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2019 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+
+using DG.Tweening;
+using UnityEngine;
+
+namespace _PartyParkade.Script.Data
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/Data/CommonAnimations.cs]: namespaces having an "_" seem a bit off to me!! Is this regular convention?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Data/CommonAnimations.cs]: You're supposed to name space the folders

**Comment by mikeslgoh** [comment]
Is it possible to narrow down the relevant files that we should be looking at? Just because I don't think we will be able to navigate through 4000 file changes

**Pull request closed without merging**